SMARTAGRIFOOD/wineTraceability/vineyardBlock/CONTRIBUTORS.YAML

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of all CONTRIBUTORS across different objects (data models) alphabetically ordered. All fields are non mandatory
```

Comment:
same as the other CONTRIBUTORS.yaml is located at the root of the subject. But the file is correct.